comp_loinc/build.py

```diff
@@ -14,6 +28,12 @@ def build_part_ontology(
         part_supplementary_file: str,
         hierarchy_file: str,
         output: str):
+    """Build part ontology.
```

Comment:
Added some tentative documentation to each command. When you run `--help` on the command on the command line, it will show the contents of this docstring.

Once I understand more about how it all works, I'll update these.